docs/src/modules/javascript/pages/views.adoc

```diff
@@ -213,6 +214,37 @@ include::example$js-customer-registry/customer_view.proto[tag=wrap-repeated]
 <2> The return type is not `streamed`.
 <3> The `repeated` field is referenced in the query with `* AS results`.
 
+[#testing]
+== How to test a View
+
+View tests need to create gRPC clients for both the Entity and the View. For example:
+
+[source,javascript]
+----
+include::example$js-customer-registry/integration-test/customer-registry-test.js[lines=17..30]
```

Comment:
Would be much better to use tags in the source files, rather than line numbers. We could update the test and never realise it's referenced from the docs.